client/components/woopay/save-user/checkout-page-save-user.js

```diff
@@ -73,7 +73,9 @@ const CheckoutPageSaveUser = ( { isBlocksCheckout } ) => {
 				? {}
 				: {
 						save_user_in_woopay: isSaveDetailsChecked,
-						woopay_source_url: window.location.href,
+						woopay_source_url: window.location.href.split(
```

Comment:
This would not work with plain permalinks, maybe we can change it to `wcSettings.storePages.checkout.permalink`.

Reply:
Great catch! Updated in https://github.com/Automattic/woocommerce-payments/pull/7924/commits/abbd2a4b3af8844c0762a3c4406cae6054540940

I wasn't sure if `storePages` would be available on all sites. Then realized that it's set in Blocks. Since Blocks is packaged with WooPayments we should be good.